tsconfig.json

```diff
@@ -14,8 +14,19 @@
 		"isolatedModules": true,
 		"jsx": "preserve",
 		"jsxImportSource": "@emotion/react",
-		"incremental": true
+		"incremental": true,
+		"plugins": [
+			{
+				"name": "next"
+			}
+		]
 	},
-	"include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".storybook/preview.tsx"],
+	"include": [
+		"**/*.ts",
+		"**/*.tsx",
+		".storybook/preview.tsx",
```

Comment:
Is this necessary?

Reply:
Next rewrites the `tsconfig` file whenever the `dev` script is run. I don't think it's necessary but I can't figure out how to prevent this.